.github/workflows/up_ex_doc_version.yml

```diff
@@ -23,9 +23,6 @@ jobs:
           - otp: '26'
             elixir: '1.15'
             rebar3: '3.22'
-          - otp: '27.0-rc1'
-            elixir: '1.16'
-            rebar3: '3.22'
```

Comment:
This was being wrongly tested as Elixir 1.16 over Erlang/OTP `rc1`. It seems `setup-beam` might need some TLC around the use of `rc`.